### ISSUE TITLE
Landing page update

### ISSUE THREAD

**Issue by Jomari-Macaraig** [pythonph/pycon-2019#3]
- Add date of the event (February 24-25, 2019)
- Add venue to the event (iACADEMY Nexus Campus, 7434 Yakal, Makati, 1203)
- Add section for Code of Conduct
- Links
   - Call for proposal
   - Call for volunteers(if already present)
   - Be a sponsor(if already present)
- About PyCon

### NOTE: 
- If possible provide a map for the venue/location

**Comment by marksteve** [comment]
@Jomari-Macaraig Are there details na for this?

**Comment by Jomari-Macaraig** [comment]
@marksteve Sorry for replying late. I''m currently gonna work with some designer(s) to have a more detailed wireframe.

I was thinking for the meantime just to have this details in the landing page so we can just improve it once the wireframes are up.

**Comment by marksteve** [comment]
@Jomari-Macaraig Yeah I wasn't looking for the designs (my girlfriend can help with this!). Was looking for the details themselves. What's the date of the event? Where's the venue? Etc.

**Comment by Jomari-Macaraig** [comment]
Hi @marksteve thanks for pointing that out :) I've updated the description, let me know if I miss something again

**Comment by Jomari-Macaraig** [comment]
Hi @marksteve just looping you with the update with the design. I've someone to work with the design :)
